src/scss/pages/_home.scss

```
@@ -104,39 +130,200 @@
           @include absolute(50%, null, null, 50%);
           @include css3-prefix(transform, translate(-50%, -50%));
         }
       }
     }
   }
+}
+
+.blog {
+  @include paddingY(10rem,9.5rem);
+
+  &__inner {
+    @include flexbox(space-between);
+
+    &-content{
+      width: 65%;
+    }
+
+    &-customer {
+      width: 35%;
+      text-align: center;
+
+      .customer__quote {
+        img {
+          margin: 3rem auto;
+        }
+      }
+
+      .customer__comment {
+        max-width: 80%;
+        margin: 0 auto;
+      }
+
+      .customer__author {
+        &-avt {
+          @include relative;
+          @include size(10rem);
+          margin: 25px auto;
+          overflow: hidden;
+          border-radius: 2rem;
+
+          img {
+            @include absolute(10%,null,null,-10%);
+            @include css3-prefix(transform,scale(1.5));
+          }
+        }
+
+        &-name {
+          color: $dark-color
+        }
+
+        &-job {
+          font-size: 1.3rem;
+          font-style: italic;
+        }
+      }
+    }
+
+    @include maxWidth($grid__bp-md) {
+      flex-direction: column;
+      &-content {
+        width: 100%;
+      }
+      &-customer {
+        width: 100%;
+      }
+    }
+  }
+
+}
+
+.blog__list {
+  @include flexbox(space-between, null,null,wrap)
+}
+
+.blog__item {
+  width: calc(50% - 2rem);
+
+  @include maxWidth($grid__bp-md) {
+    width: 100%;
+    margin-bottom: 2.5rem
+  }
+
+  &-image {
+    @include relative;
+    width: 100%;
+    margin-bottom: 2.5rem;
+    padding-top: (9 / 16) * 100%;
+    overflow: hidden;
+
+    &:hover i {
+      @include css3-prefix(transform, translate(-50%, -50%) rotate(90deg));
+      opacity: 1;
+    }
+
+    &:hover::after {
+      @include css3-prefix(transform, rotate(-29.3deg) translateY(0));
+    }
+
+    &::after {
+      content: "";
+      @include absolute(8%, null, null, 0);
+      @include size(200%,100%);
+      background-color: rgba(255, 255, 255, 0.5);
+      @include css3-prefix(transform, rotate(-29.3deg) translateY(90%));
+      transition: all 0.3s;
+    }
+
+    i {
+      @include absolute(50%, null, null, 50%);
+      padding: 1.5rem;
+      border-radius: 50%;
+      font-size: 1.5rem;
+      background-color: rgba(255, 255, 255, 0.9);
+      @include css3-prefix(transform, rotate(180deg) translate(-50%, -50%));
+      transition: all .3s;
+      transition-delay: .3s;
+      opacity: 0;
+      z-index: 99;
+    }
+
+    img {
+      @include absolute(0,null,null,0);
+      @include size(100%);
+    }
+  }
+
+  &-title {
+    font-size: 1.8rem;
+    letter-spacing: 0.1rem;
+    color: $dark-color;
+  }
+
+  &-info {
+    margin-bottom: 1rem;
+    font-size: 1.2rem;
+    color:$gray-light-color;
+  }
+
+  &-desc {
+    display: block;
+    display: -webkit-box;
+    max-width: 400px;
+    height: 14px*1.5*3;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    text-overflow: ellipsis;

```

Cái giới hạn dòng này nên tạo 1 mixin riêng, để có thể tái sử dụng ở nhiều chỗ khác nữa
```suggestion
 @mixin limitLine($line) {
    -webkit-line-clamp: $line;
    -webkit-box-orient: vertical;
    overflow: hidden;
    text-overflow: ellipsis;
 }
```